### PR TITLE
Added functionality to remember last selected Electorate

### LINKED ISSUE #1: Remember last selected electorate and redirect there when landing on home page late
From https://www.reddit.com/r/AustralianPolitics/comments/umwy7g/comment/i8aknc1/?utm_source=share&utm_medium=web2x&context=3:

> Persist the filled in ballot papers as a landing page. (When I return to the website I have to select the electorate again when this is unnecessary if previously chosen)

We can save the last selected electorate in local storage. The tricky bit might be recognizing if the user is going to the home page by clicking the navigation (in that case we do *not* want to go back to the electorate), vs the user coming to the landing page through a fresh page load.

### PR DESCRIPTION
**Pull request by naiknareshh**
Fixes #1

### CODE REVIEW

File: src/routes/home/index.tsx [hunk 1]
```diff
@@ -1,8 +1,18 @@
...
 import style from "./style.css";
 import SelectElectorate from "../../components/select-electorate";
+import { useEffect } from "preact/hooks";
+import { route } from "preact-router";
 
 const Home: FunctionalComponent = () => {
+
+  useEffect(() => {
+    let selectedElectorate = localStorage.getItem('electorate')
+    if(selectedElectorate){
```
Comment (robinst) [inline src/routes/home/index.tsx]: Can you run prettier on the files you changed (`npx prettier --write .`)? It should fix the formatting, e.g. here and the line above (missing semicolon).

File: src/components/select-electorate/index.tsx [hunk 2]
```diff
@@ -48,6 +48,7 @@ const groupedOptions = () => {
 const onChange = (option: Option | null) => {
   if (option !== null) {
     route(`/electorate/${option.value}`);
+    localStorage.setItem('electorate', option.value);
   } else {
     route("/");
   }
```
Comment (robinst) [inline src/components/select-electorate/index.tsx:55]: In the else case here we probably also want to clear the stored electorate, right?

**Review by robinst — CHANGES_REQUESTED** [review]
Nice, haven't tested it yet but looks promising. I added some inline comments to fix.

File: src/components/select-electorate/index.tsx (hunk 2 above)
Comment (naiknareshh) [reply src/components/select-electorate/index.tsx:55]: Right, added code to remove from localStorage in the else block

File: src/routes/home/index.tsx (hunk 1 above)
Comment (naiknareshh) [reply src/routes/home/index.tsx]: This is done in the new commit

**Comment by robinst** [comment]
Thanks, works nicely!

**Pull request merged**